app.py

```diff
@@ -104,23 +111,24 @@ def __init__(self):
         self.public_key_bytes = self.public_key.public_bytes(
             encoding=serialization.Encoding.Raw, format=serialization.PublicFormat.Raw
         )
+        self.model_list = self.model_config.find_one({"name": "model_list"})["data"]
         self.message = "image-generating-subnet"
         self.signature = base64.b64encode(
             self.private_key.sign(self.message.encode("utf-8"))
         )
 
         self.loop = asyncio.get_event_loop()
 
-        self.app.add_api_route(
-            "/get_credentials", self.get_credentials, methods=["POST"]
-        )
-        self.app.add_api_route("/generate", self.generate, methods=["POST"])
-        self.app.add_api_route("/get_validators", self.get_validators, methods=["GET"])
-        self.app.add_api_route("/api/v1/txt2img", self.txt2img_api, methods=["POST"])
-        self.app.add_api_route("/api/v1/img2img", self.img2img_api, methods=["POST"])
-        self.app.add_api_route(
-            "/api/v1/instantid", self.instantid_api, methods=["POST"]
-        )
+        # self.app.add_api_route(
```

Comment:
Should you clean the code by removing these commented codes?

Reply:
Sure, I will.